traveler-ios-ui/TravelerKitUI/ViewControllers/AvailabilityCalendarResultViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  AvailabilityCalendarResultViewController.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-10-28.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import UIKit
+import TravelerKit
+
+private let edgeInset = 10
+private let cellSpace = 5
+let calendarDateCellIdentifier = "calendarDateCellIdentifier"
+
+protocol AvailabilityCalendarResultViewControllerDelegate: class {
+    func availabilityCalendarResultViewController(_ controller: AvailabilityCalendarResultViewController, didSelect availability: Availability)
+}
+
+class AvailabilityCalendarResultViewController: UICollectionViewController {
+    var representingDate: Date?
+    var availabilities: [Availability]?
+    var delegate: AvailabilityCalendarResultViewControllerDelegate?
```

Comment:
delegates MUST all be WEAK...this creates a memory leak.

Please read up on why this is the case